<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsModelComponent</class>
 <widget class="QFrame" name="CSettingsModelComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Models settings</string>
  </property>
  <layout class="QHBoxLayout" name="hl_SettingsModelComponent">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QCheckBox" name="cb_AllowExcludeModels">
     <property name="toolTip">
      <string>Hint: This can cause wrong behaviour or violate 3rd parties copyright. You need to know what you are doing enabling the feature.</string>
     </property>
     <property name="text">
      <string>allow &quot;excluded&quot; models in set</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="hs_SetiingsModelComponent">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Consolidate">
     <property name="text">
      <string>Consolidate (secs):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_ConsolidateSecs">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="placeholderText">
      <string>secs.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="comp_Led" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::CLedWidget</class>
   <extends>QWidget</extends>
   <header>gui/led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
